### DISCUSSION TITLE
bash api call get * with filter user_id = auth.uid()

### DISCUSSION THREAD

**Discussion by amaxrich** [supabase/supabase#19001] [Questions]
What the bash command for api call get * with filter user_id = auth.uid()? 
Base url is: http://myproject.supabase.co/rest/v1/user_event_view

**Comment by GaryAustin1** [comment]
Your question is not very clear.  If you use a filter from a client there is no auth.uid() function.  Auth.uid() is only used in RLS or in postgres functions not in a filter.   Normally you would use the user id from the session on the client in a filter and also back it up with RLS of user_id = auth.uid() in policies to insure the user id is real.
